src/OrleansGCPUtils/Providers/Streams/PubSub/IPubSubDataAdapter.cs

```diff
@@ -7,12 +7,12 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace Orleans.Providers.Streams
+namespace Orleans.Providers.GCP.Streams.PubSub
 {
     /// <summary>
     /// Converts event data to and from cloud queue message
     /// </summary>
-    public interface IGooglePubSubDataAdapter
```

Comment:
Why do we remove Google from the naming? It is easy for people to relate this with Orleans streaming pubsub store, even though it is not that at all. It can be confusing.